app/assets/stylesheets/global/html_forms/elmo_forms/_general.scss

```diff
@@ -33,4 +33,10 @@
     font-weight: bold;
     margin-bottom: 5px;
   }
+
+  .submit-buttons {
+    .float-right {
+      margin-left: 3px;
+    }
+  }
```

Comment:
This is a clever solution! However, instead of float, could we use flex order? It has less side effects, I think, than float:

https://getbootstrap.com/docs/4.0/utilities/flex/#order

Reply:
Adding `d-flex` makes the buttons left-aligned and removes some default padding between them, which needs to be manually undone as needed... so to support button ordering in a reusable way I'll add styles via SCSS instead of using Bootstrap, and test to make sure existing buttons work fine (including with "upload progress" elements in some cases). There are 17 existing `.submit-buttons` that I want to be careful about, but flexbox will make them all consistent instead of just here 🙌

Reply:
Done and tested in RTL language as well